.jenkins/pytorch/win-test-helpers/build_pytorch.bat

```diff
@@ -37,33 +37,19 @@ if "%VC_VERSION%" == "" (
 @echo on
 popd
 
-if "%CUDA_VERSION%" == "9" goto cuda_build_9
-if "%CUDA_VERSION%" == "10" goto cuda_build_10
-if "%CUDA_VERSION%" == "11" goto cuda_build_11
-goto cuda_build_end
+if not "%USE_CUDA%"=="1" goto cuda_build_end
 
-:cuda_build_9
+set CUDA_PATH=C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v%CUDA_COMPLETE_VERSION%
 
-set CUDA_PATH=C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v9.2
-set CUDA_PATH_V9_2=%CUDA_PATH%
+rem version transformer, for example 10.1 to 10_1. But I don't find where the CUDA_PATH_VXX_X is used 
+set VERSION_SUFFIX=%CUDA_COMPLETE_VERSION:.=_%
+set CUDA_PATH_V%VERSION_SUFFIX%=CUDA_PATH
```

Comment:
Well, are you sure this line is correct?

Reply:
Good Catch. it should _set CUDA_PATH_V%VERSION_SUFFIX%=%CUDA_PATH%_

Reply:
https://circleci.com/api/v1.1/project/github/pytorch/pytorch/9567605/output/104/0?file=true&allocation-id=5fd480fbf1f454400cd69497-0-build%2F12B49615

```
(base) circleci@PACKER-5FD0AAF5 C:\Users\circleci\project>rem version transformer, for example 10.1 to 10_1. 
(base) circleci@PACKER-5FD0AAF5 C:\Users\circleci\project>set VERSION_SUFFIX=10_1 
(base) circleci@PACKER-5FD0AAF5 C:\Users\circleci\project>set CUDA_PATH_V10_1=C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v10.1 
(base) circleci@PACKER-5FD0AAF5 C:\Users\circleci\project>set CUDNN_LIB_DIR=C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v10.1\lib\x64
```